cmake/kokkos_arch.cmake

```diff
@@ -98,9 +98,10 @@ IF(KOKKOS_ENABLE_COMPILER_WARNINGS)
     ${COMMON_WARNINGS})
 
   COMPILER_SPECIFIC_FLAGS(
-    PGI NO-VALUE-SPECIFIED
-    GNU     ${GNU_WARNINGS}
-    DEFAULT ${COMMON_WARNINGS}
+    COMPILER_ID CMAKE_CXX_COMPILER
```

Comment:
Slightly confused by this. Shouldn't this be `CMAKE_CXX_COMPILER_ID` ?

Reply:
Thanks, fixed!